tests/test_lib_tokens_tiqr.py

```diff
@@ -684,7 +684,7 @@ def test_05_api_endpoint_with_multiple_tokens(self):
                         "operation": "login"}
         r = TiqrTokenClass.api_endpoint(req, g)
         self.assertEqual(r[0], "plain")
-        self.assertEqual(r[1], "INVALID_RESPONSE")
+        self.assertRegexpMatches(r[1], r"INVALID_RESPONSE:[0-9]+")
```

Comment:
I think, we should also test, that the counter is decreased correctly. I can provide a test for that since this might be a bit tricky.

Reply:
@plettich Would you enhance the test in a seperate pull request?

Reply:
will do.